package.json

```diff
@@ -33,6 +33,10 @@
     "test:e2e:storybook:ci": "run-p --race storybook cypress:storybook",
     "test:e2e:storybook:interactive": "npm run killStorybook && run-p --race storybook cypress:storybook:interactive",
     "test:lighthouse": "npm run killApp && npm run build && run-p --race start lighthouse",
+    "test:a11y-tests-web:bbc-a11y" : "wait-on http://localhost:7080/ && A11Y_CONFIG=simorgh/simorgh A11Y_SIMORGH_BASE_URL=http://localhost:7080 npm --prefix ./node_modules/@bbc/a11y-tests-web run start:bbc-a11y:headless",    
+    "test:a11y-tests-web:bbc-a11y:ci" : "npm run build && run-p --race start test:a11y-tests-web:bbc-a11y",
+    "test:a11y-tests-web:lighthouse": "wait-on http://localhost:7080/ &&  A11Y_CONFIG=simorgh/simorgh OUTPUT_JSON='/../../../../reports/lighthouse-report.json' npm --prefix ./node_modules/@bbc/a11y-tests-web run start:lighthouse:junit",
```

Comment:
This `/../../../../` is hard to read. Want to use `pwd` instead, but unclear how to pass this as a parameter.